source/installation/vm/tangobox.rst

```diff
@@ -0,0 +1,367 @@
+
+   .. figure:: tangobox/import-2.png
+
+      A virtual machine settings window.
+
+      You may change number of CPUs and increase RAM size. Memory size has major impact on VM performance.
+
+* Wait for VirtualBox to import machine
+
+After importing the VM image to VirtualBox you may start it.
+
+* Username is: `tango-cs`
+* Password is: `tango`
+
+`tango-cs` user has sudo rights, so he may invoke commands as superuser with command :command:`sudo`.
+
+You may explore the Tango Controls feature by clicking related shortcuts on the Desktop.
+
+.. note::
+
+   Plese note that some shortcuts are related to features running on containers. Please start related container first.
+   See the following section.
+
+
+.. _container_switch_on_off:
+
+Switching containers on and off
+-------------------------------
+
+Some of the features of Tango are provided inside pre-build docker containers.
+These can be switched on and off by starting or stopping related containers.
+Containers behave similar to virtual machines with they own network cards and operating system stack,
+however, lacking full separations.
+
+To start a container, open terminal and invoke :command:`docker start {container-name}`. For example, to star
+a linac simulation use the following statement:
+
+.. code-block:: bash
+
+   docker start tangobox-sim
+
+
+To stop a container, open terminal and invoke :command:`docker stop {container-name}`. For example, to stop
+a linac simulation use the following statement:
+
+.. code-block:: bash
+
+   docker stop tangobox-sim
+
+To see which containers are running please, call :command:`docker ps`
+
+
+Deployment structure
+--------------------
+
+Network
+~~~~~~~
+
+Containers are created withing their own subnet: `172.18.0.0/16`. The network is called `tango_nw`
+The subnet was created with the following docker command:
+
+.. code-block:: bash
+
+   docker network create --driver=bridge --subnet=172.18.0.0/16 --opt com.docker.network.bridge.enable_icc=true \
+   --opt com.docker.network.bridge.host_binding_ipv4="0.0.0.0" --opt com.docker.network.bridge.mtu=1500 \
+   --opt com.docker.network.bridge.enable_ip_masquerade=true  tango_nw
+
+Containers are assigned static IPs. List of the IPs assignment maybe seen in :file:`/etc/hosts`. Use command
+:command:`cat /etc/hosts` to see its contents.
+
+Containers and images dependency
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Each container is based on its image. All images are already build but, if neccessary, *Dockerfiles* are stored in :file:`home/Dockerfiles`
+directory. Below is the list of all containers and corresponding images:
+
+================== =========== ===============================
+Container            Image             Remarks
+================== =========== ===============================
+tangobox-com         com                 -
+tangobox-sim         sim                 -
+tangobox-archiving   archive              -
+tangobox-hdbpp       hdbpp               -
```

Comment:
The images are hosted in a registry, so above image name is no longer valid. See:

```
CONTAINER ID        IMAGE                                                                        COMMAND                  CREATED             STATUS              PORTS               NAMES
04e048c14dba        registry.gitlab.com/s2innovation/tangobox-docker/tangobox-hdbpp:latest       "/bin/sh -c '/usr/bi…"   12 days ago         Up 8 minutes                            tangobox-hdbpp
```